test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9,14 +9,21 @@
 
 import numpy as np
 import io
+import itertools
 
+from torch.nn.utils import rnn as rnn_utils
+from model_defs.lstm_flattening_result import LstmFlatteningResult
+from model_defs.rnn_model_with_packed_sequence import RnnModelWithPackedSequence
 from test_pytorch_common import skipIfUnsupportedMinOpsetVersion, skipIfUnsupportedOpsetVersion
-
-
 import model_defs.word_language_model as word_language_model
 
 
-def run_model_test(self, model, train, batch_size=2, state_dict=None,
+RNN_BATCH_SIZE = 7
```

Comment:
add and import these variables from test_pytorch_common to avoid duplicating them with the ones in the caffe2 file